### PR TITLE
Don’t set active when disabled

### LINKED ISSUE #24: Disabled button gets active state on Safari
Accessing a disabled button adds an "active" attribute to the button on Safari (works on Chrome).

### PR DESCRIPTION
**Pull request by tomivirkki**
Fixes #24 

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-button/23)

### CODE REVIEW

**Comment by platosha** [comment]
:lgtm:

---

Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-button/23#-:-KkpQzzhOTx3c_U7QmGh:bnfp4nl)*

File: test/vaadin-button_test.html [hunk 1]
```diff
@@ -100,6 +100,16 @@
 
         expect(vaadinButton.hasAttribute('active')).to.be.false;
       });
+
+      it('should not have active attribute when disabled', () => {
```
Comment (limonte) [inline test/vaadin-button_test.html]: `'should not have active attribute' => 'should not get an active attribute form keyboard events'`

**Comment by tomivirkki** [comment]
*[test/vaadin-button_test.html, line 104 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-button/23#-KkpPjFk-52upDmicGYL-r1-104:-KkpT4Y2SZclNCC_dSQO:brrxray) ([raw file](https://github.com/vaadin/vaadin-button/blob/2bd1a1568732e808f025a316bfafd66bc40eaefd/test/vaadin-button_test.html#L104)):*
<details><summary><i>Previously, limonte (Limon Monte) wrote…</i></summary><blockquote>

`'should not have active attribute' => 'should not get an active attribute form keyboard events'`
</blockquote></details>

...It's not just for keyboard events.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-button/23)*

**Comment by tomivirkki** [comment]
*[test/vaadin-button_test.html, line 104 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-button/23#-KkpPjFk-52upDmicGYL-r1-104:-KkpTM8vNxbJU22Zbj3U:bdq63x) ([raw file](https://github.com/vaadin/vaadin-button/blob/2bd1a1568732e808f025a316bfafd66bc40eaefd/test/vaadin-button_test.html#L104)):*
<details><summary><i>Previously, tomivirkki (Tomi Virkki) wrote…</i></summary><blockquote>

...It's not just for keyboard events.
</blockquote></details>

...and the wording "should not have active attribute..." derives from the prior tests

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-button/23)*

File: test/vaadin-button_test.html (hunk 1 above)
Comment (manolo) [inline test/vaadin-button_test.html]: Seems that this comment was not shown in reviewable, hence this was ignored.

Comment (limonte) [inline test/vaadin-button_test.html]: https://github.com/vaadin/vaadin-button/pull/23#issuecomment-303395845

**Pull request merged**